src/meta/processors/admin/DropSnapshotProcessor.cpp

```diff
@@ -20,6 +20,11 @@ void DropSnapshotProcessor::process(const cpp2::DropSnapshotReq& req) {
   auto ret = doGet(std::move(key));
   if (!nebula::ok(ret)) {
     auto retCode = nebula::error(ret);
+    if (retCode == nebula::cpp2::ErrorCode::E_KEY_NOT_FOUND) {
```

Comment:
if req.name() is a wrong snapshot name, would it return the error "snapshot already dropped"? it may confuse the user

Reply:
I'll change it